### PR TITLE
PHP 8.0 support

### PR DESCRIPTION
**Pull request by Neur0toxine**

### CODE REVIEW

**Comment by dbu** [comment]
thanks for the pull request.

i notice that travis-ci seems to no longer run, so we don't see if things succeed in PHP 8. i fear we need to switch to github actions to get a build again. not sure if i get to work on that anytime soon... if you have some time available to do that chore, it would be appreciated ;-)

**Comment by Neur0toxine** [comment]
Done.

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "phpstan/phpstan-phpunit": "^0.12",
         "phpunit/phpunit": "^8.0",
         "friendsofphp/php-cs-fixer": "^2.14",
-        "jms/serializer": "^1.13 || ^2"
+        "jms/serializer": "^2 || ^3"
```
Comment (dbu) [inline composer.json:31]: could we keep allowing 1.13 or is there a problem with that version?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+  test:
+    name: "PHPUnit"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.2', '7.3', '7.4', '8.0']
+        include:
+          - php-version: '7.3'
```
Comment (dbu) [inline .github/workflows/ci.yml:19]: why not 7.2 for the lowest version build?

File: composer.json (hunk 1 above)
Comment (Neur0toxine) [reply composer.json:31]: No, it's not possible. 1.13 does not support PHP 8.x and was dropped from the `liip/metadata-parser`.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Neur0toxine) [reply .github/workflows/ci.yml]: It's a mistake; fixed.

**Comment by dbu** [comment]
thanks a lot!

i pushed it as a branch in this repository to get github to pick up the action, and merged #25.

i will tag a release now.

**Pull request closed without merging**